src/core/transposition.h

```diff
@@ -140,8 +174,8 @@ class TranspositionTable {
         /* only update entry if a better one is found */
         if (!sameKey
             || entryData.move.isNull()
-            || depth >= entryData.depth
-            || (flag == TtExact && entryData.flag != TtExact)) {
+            || depth + (2 * ttPv) >= entryData.depth
```

Comment:
Should this be a multiplication or a bitshift?

Reply:
I did some initial research and I didn't find any engine that didn't use 2 here. 
It might actually be interesting to add this as a SPSA parameter instead and let the tuner have a go at it! But maybe we should leave that to a tuning build (hoping to do that after 1.8) :)

Reply:
Sounds good, I think that could be an interesting follow-up - mostly because I'm curious to know what the 'optimal' value might be here :)